pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -347,6 +355,60 @@ public void testInvertedIndexTriggering()
         throw new RuntimeException(e);
       }
     }, 600_000L, "Failed to generate inverted index");
+
+    final long tableSizeWithNewIndex = getTableSize(getTableName());
```

Comment:
(code format) we don't usually put `final` for local variables

Reply:
good catch!